Failed to call service /gazebo_objects/get_info

Hello,

I'm trying to tinker around with the fake_object_recognition and tried to execute [the tutorial ](https://github.com/JenniferBuehler/gazebo-pkgs/wiki/Object-information-and-recognition-tutorial). Until step 3 (spawning the cube) everything works perfectly. At step 4 I get this error:

rosuser@computer ~/test_jaco $ rosrun gazebo_state_plugins gazebo_request_object_info cube1
[ INFO] [1479977436.038761419]: Got objects topic name: </gazebo_objects/get_info>
[ERROR] [1479977436.039658131]: Failed to call service /gazebo_objects/get_info, success flag: 0

Actually the service is not running:
rosuser@computer ~/test_jaco $ rosservice list
/fake_object_recognizer/get_loggers
/fake_object_recognizer/set_logger_level
/gazebo/apply_body_wrench
/gazebo/apply_joint_effort
/gazebo/clear_body_wrenches
/gazebo/clear_joint_forces
/gazebo/delete_model
/gazebo/get_joint_properties
/gazebo/get_link_properties
/gazebo/get_link_state
/gazebo/get_loggers
/gazebo/get_model_properties
/gazebo/get_model_state
/gazebo/get_physics_properties
/gazebo/get_world_properties
/gazebo/pause_physics
/gazebo/reset_simulation
/gazebo/reset_world
/gazebo/set_joint_properties
/gazebo/set_link_properties
/gazebo/set_link_state
/gazebo/set_logger_level
/gazebo/set_model_configuration
/gazebo/set_model_state
/gazebo/set_parameters
/gazebo/set_physics_properties
/gazebo/spawn_gazebo_model
/gazebo/spawn_sdf_model
/gazebo/spawn_urdf_model
/gazebo/unpause_physics
/gazebo_test_tools/recognize_object
/object_tf_broadcaster/get_loggers
/object_tf_broadcaster/register_object
/object_tf_broadcaster/set_logger_level
/robot_state_publisher/get_loggers
/robot_state_publisher/set_logger_level
/rosout/get_loggers
/rosout/set_logger_level
/rqt_gui_py_node_7161/get_loggers
/rqt_gui_py_node_7161/set_logger_level

I'm running ROS Jade with Gazebo 5.3.0 on Mint based on Ubuntu 14.04, I checked out convenience-pkgs, gazebo-pkgs, gazebo-pkgs, jaco-arm-pkgs, joint-control-pkgs and the dependencies from the wiki. Compilation works fine and I'm able to start the gazebo simulation with the robot. Therefore I assume that something is wrong with loading the world plugin or the later loaded plugins that offer the get_info service...

Since I'm quite new to Gazebo and it's plugin system a couldn't figure out until now, where the chain is broken. Any suggestions?

--- a/jaco_tutorial/jaco_on_table/launch/jaco_on_table_gazebo_controlled.launch
+++ b/jaco_tutorial/jaco_on_table/launch/jaco_on_table_gazebo_controlled.launch
@@ -33,7 +33,7 @@
     # load the world plugin loader (in package gazebo_world_plugin_loader) which launches several
     # world plugins according to the specification in the ROS parameters. Note that you will
     # have to load those parameters on the ROS parameter server BEFORE gazebo gets launched. 
-    <arg name="load_world_plugins" default="false"/>
+    <arg name="load_world_plugins" default="true"/>
 
     # set to true if a Joint::SetVelocity() is to be used instead of
     # Joint::SetForce(). Note that this will require loading different PID values.

